dnf: consistenty use -y

bastelfreak opened the pull request:
The installation guidelines use `-y` for all dnf/yum operations, except for enabling the module. DNF support `-y` here as well, so we can use it.

bastelfreak committed:
dnf module: Use -y option

The installation guidelines use `-y` for all dnf/yum operations, except
for enabling the module. DNF support `-y` here as well, so we can use
it.

evgeni commented on _includes/manuals/3.7/3.3.1_rpm_packages.md:
```diff
@@ -27,7 +27,7 @@ To set up the repository, foreman-release packages are provided which add a repo
 
     # EL8
     dnf install https://yum.theforeman.org/releases/{{page.version}}/el8/x86_64/foreman-release.rpm
```
this install is not using `-y`, so using `-y` in the line below seems odd

bastelfreak replied:
```
bastelfreak@bastelfreak-nb ~/code/theforeman.org $ git grep 'dnf install https:'
_includes/manuals/3.4/3.3.1_rpm_packages.md:    dnf install https://yum.theforeman.org/releases/{{page.version}}/el8/x86_64/foreman-release.rpm
_includes/manuals/3.5/3.3.1_rpm_packages.md:    dnf install https://yum.theforeman.org/releases/{{page.version}}/el8/x86_64/foreman-release.rpm
_includes/manuals/3.6/3.3.1_rpm_packages.md:    dnf install https://yum.theforeman.org/releases/{{page.version}}/el8/x86_64/foreman-release.rpm
_includes/manuals/3.7/3.3.1_rpm_packages.md:    dnf install https://yum.theforeman.org/releases/{{page.version}}/el8/x86_64/foreman-release.rpm
_includes/manuals/3.8/3.3.1_rpm_packages.md:    dnf install https://yum.theforeman.org/releases/{{page.version}}/el8/x86_64/foreman-release.rpm
_includes/manuals/nightly/3.3.1_rpm_packages.md:    dnf install https://yum.theforeman.org/releases/{{page.version}}/el8/x86_64/foreman-release.rpm
bastelfreak@bastelfreak-nb ~/code/theforeman.org $ git grep 'dnf -y install https:'
_includes/manuals/1.12/2.1_quickstart_installation.md:dnf -y install https://yum.theforeman.org/releases/{{page.version}}/f24/x86_64/foreman-release.rpm
_includes/manuals/1.13/2.1_quickstart_installation.md:dnf -y install https://yum.theforeman.org/releases/{{page.version}}/f24/x86_64/foreman-release.rpm
_includes/manuals/1.14/2.1_quickstart_installation.md:dnf -y install https://yum.theforeman.org/releases/{{page.version}}/f24/x86_64/foreman-release.rpm
_includes/manuals/1.15/2.1_quickstart_installation.md:dnf -y install https://yum.theforeman.org/releases/{{page.version}}/f24/x86_64/foreman-release.rpm
_includes/manuals/3.3/2.1_quickstart_installation.md:sudo dnf -y install https://yum.puppet.com/puppet7-release-el-8.noarch.rpm
_includes/manuals/3.3/2.1_quickstart_installation.md:sudo dnf -y install https://yum.theforeman.org/releases/{{page.version}}/el8/x86_64/foreman-release.rpm
_includes/manuals/3.4/2.1_quickstart_installation.md:sudo dnf -y install https://yum.puppet.com/puppet7-release-el-8.noarch.rpm
_includes/manuals/3.4/2.1_quickstart_installation.md:sudo dnf -y install https://yum.theforeman.org/releases/{{page.version}}/el8/x86_64/foreman-release.rpm
_includes/manuals/3.5/2.1_quickstart_installation.md:sudo dnf -y install https://yum.puppet.com/puppet7-release-el-8.noarch.rpm
_includes/manuals/3.5/2.1_quickstart_installation.md:sudo dnf -y install https://yum.theforeman.org/releases/{{page.version}}/el8/x86_64/foreman-release.rpm
_includes/manuals/3.6/2.1_quickstart_installation.md:sudo dnf -y install https://yum.puppet.com/puppet7-release-el-8.noarch.rpm
_includes/manuals/3.6/2.1_quickstart_installation.md:sudo dnf -y install https://yum.theforeman.org/releases/{{page.version}}/el8/x86_64/foreman-release.rpm
_includes/manuals/3.7/2.1_quickstart_installation.md:sudo dnf -y install https://yum.puppet.com/puppet7-release-el-8.noarch.rpm
_includes/manuals/3.7/2.1_quickstart_installation.md:sudo dnf -y install https://yum.theforeman.org/releases/{{page.version}}/el8/x86_64/foreman-release.rpm
_includes/manuals/3.8/2.1_quickstart_installation.md:sudo dnf -y install https://yum.puppet.com/puppet7-release-el-8.noarch.rpm
_includes/manuals/3.8/2.1_quickstart_installation.md:sudo dnf -y install https://yum.theforeman.org/releases/{{page.version}}/el8/x86_64/foreman-release.rpm
_includes/manuals/nightly/2.1_quickstart_installation.md:sudo dnf -y install https://yum.puppet.com/puppet7-release-el-8.noarch.rpm
_includes/manuals/nightly/2.1_quickstart_installation.md:sudo dnf -y install https://yum.theforeman.org/releases/{{page.version}}/el8/x86_64/foreman-release.rpm
scripts/installer/Dockerfile:RUN dnf -y install https://yum.puppetlabs.com/puppet7/puppet7-release-el-8.noarch.rpm
scripts/installer/Dockerfile:RUN dnf -y install https://yum.theforeman.org/releases/$FOREMAN_VERSION/el8/x86_64/foreman-release.rpm
bastelfreak@bastelfreak-nb ~/code/theforeman.org $
```

I will fix that in another commit

bastelfreak committed:
foreman-release.rpm install: Always use -y

This was a bit inconsistent across the different versions:

```
bastelfreak@bastelfreak-nb ~/code/theforeman.org $ git grep 'dnf install https:'
_includes/manuals/3.4/3.3.1_rpm_packages.md:    dnf install https://yum.theforeman.org/releases/{{page.version}}/el8/x86_64/foreman-release.rpm
_includes/manuals/3.5/3.3.1_rpm_packages.md:    dnf install https://yum.theforeman.org/releases/{{page.version}}/el8/x86_64/foreman-release.rpm
_includes/manuals/3.6/3.3.1_rpm_packages.md:    dnf install https://yum.theforeman.org/releases/{{page.version}}/el8/x86_64/foreman-release.rpm
_includes/manuals/3.7/3.3.1_rpm_packages.md:    dnf install https://yum.theforeman.org/releases/{{page.version}}/el8/x86_64/foreman-release.rpm
_includes/manuals/3.8/3.3.1_rpm_packages.md:    dnf install https://yum.theforeman.org/releases/{{page.version}}/el8/x86_64/foreman-release.rpm
_includes/manuals/nightly/3.3.1_rpm_packages.md:    dnf install https://yum.theforeman.org/releases/{{page.version}}/el8/x86_64/foreman-release.rpm
bastelfreak@bastelfreak-nb ~/code/theforeman.org $ git grep 'dnf -y install https:'
_includes/manuals/1.12/2.1_quickstart_installation.md:dnf -y install https://yum.theforeman.org/releases/{{page.version}}/f24/x86_64/foreman-release.rpm
_includes/manuals/1.13/2.1_quickstart_installation.md:dnf -y install https://yum.theforeman.org/releases/{{page.version}}/f24/x86_64/foreman-release.rpm
_includes/manuals/1.14/2.1_quickstart_installation.md:dnf -y install https://yum.theforeman.org/releases/{{page.version}}/f24/x86_64/foreman-release.rpm
_includes/manuals/1.15/2.1_quickstart_installation.md:dnf -y install https://yum.theforeman.org/releases/{{page.version}}/f24/x86_64/foreman-release.rpm
_includes/manuals/3.3/2.1_quickstart_installation.md:sudo dnf -y install https://yum.puppet.com/puppet7-release-el-8.noarch.rpm
_includes/manuals/3.3/2.1_quickstart_installation.md:sudo dnf -y install https://yum.theforeman.org/releases/{{page.version}}/el8/x86_64/foreman-release.rpm
_includes/manuals/3.4/2.1_quickstart_installation.md:sudo dnf -y install https://yum.puppet.com/puppet7-release-el-8.noarch.rpm
_includes/manuals/3.4/2.1_quickstart_installation.md:sudo dnf -y install https://yum.theforeman.org/releases/{{page.version}}/el8/x86_64/foreman-release.rpm
_includes/manuals/3.5/2.1_quickstart_installation.md:sudo dnf -y install https://yum.puppet.com/puppet7-release-el-8.noarch.rpm
_includes/manuals/3.5/2.1_quickstart_installation.md:sudo dnf -y install https://yum.theforeman.org/releases/{{page.version}}/el8/x86_64/foreman-release.rpm
_includes/manuals/3.6/2.1_quickstart_installation.md:sudo dnf -y install https://yum.puppet.com/puppet7-release-el-8.noarch.rpm
_includes/manuals/3.6/2.1_quickstart_installation.md:sudo dnf -y install https://yum.theforeman.org/releases/{{page.version}}/el8/x86_64/foreman-release.rpm
_includes/manuals/3.7/2.1_quickstart_installation.md:sudo dnf -y install https://yum.puppet.com/puppet7-release-el-8.noarch.rpm
_includes/manuals/3.7/2.1_quickstart_installation.md:sudo dnf -y install https://yum.theforeman.org/releases/{{page.version}}/el8/x86_64/foreman-release.rpm
_includes/manuals/3.8/2.1_quickstart_installation.md:sudo dnf -y install https://yum.puppet.com/puppet7-release-el-8.noarch.rpm
_includes/manuals/3.8/2.1_quickstart_installation.md:sudo dnf -y install https://yum.theforeman.org/releases/{{page.version}}/el8/x86_64/foreman-release.rpm
_includes/manuals/nightly/2.1_quickstart_installation.md:sudo dnf -y install https://yum.puppet.com/puppet7-release-el-8.noarch.rpm
_includes/manuals/nightly/2.1_quickstart_installation.md:sudo dnf -y install https://yum.theforeman.org/releases/{{page.version}}/el8/x86_64/foreman-release.rpm
scripts/installer/Dockerfile:RUN dnf -y install https://yum.puppetlabs.com/puppet7/puppet7-release-el-8.noarch.rpm
scripts/installer/Dockerfile:RUN dnf -y install https://yum.theforeman.org/releases/$FOREMAN_VERSION/el8/x86_64/foreman-release.rpm
bastelfreak@bastelfreak-nb ~/code/theforeman.org $
```

I will fix that in another PR
